frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/block_action_option_cubit.dart

```diff
@@ -29,6 +28,8 @@ class BlockActionOptionCubit extends Cubit<BlockActionOptionState> {
         break;
       case OptionAction.duplicate:
         await _duplicateBlock(transaction, node);
+        EditorNotification.paste().post();
```

Comment:
Very curious about this. Is duplicate the same as copy and paste? Why wasn't this necessary before?

Reply:
This will let the transaction service know, that a paste is made, which will in turn let the handler for eg. child pages know, that there was a paste, it's because the same way to handly copy+paste for a handler is the same as handling the duplicate (same behavior).